### ISSUE TITLE
Spinner(s) not working properly after changing the value programmatically

### ISSUE THREAD

**Issue by Rumtipper** [shaack/bootstrap-input-spinner#98]
I have several number inputs, all bound to bootstrap-input-spinner. Now I want to set their values all at once by changing a master value. This works fine.

But the inputs are not working properly afterwards; they start counting from their original values instead from the current ones set programmatically. I have noticed this only happens, if the values of multiple fields are set through one jQuery selector instead of changing each one individually. Only the first in the set of matched elements works properly.

Please have a look at this fiddle: https://jsfiddle.net/ey4sp9wu/

It works as expected on the native input.

_bootstrap-input-spinner 2.0.9_

**Comment by rbravo86** [comment]
Why don't you just use a  .each to initialize them ? 

Base on your code : 
```
// Example B: One line to change them all...
$(document).on('change', 'input#source_b', function(e) {
	$('.box_b input').each( function() { 
  	   $(this).val( $(e.target).val() );
        });
})
```

**Comment by shaack** [comment]
@rbravo86 Yes, this is a good workaround, but @Rumtipper is right, it should also work for all elements with .val().

**Comment by Rumtipper** [comment]
@rbravo86 Yes, I had considered that, thank you. However, the "regular" solution looks cleaner to me (I don't know about performance differences, but me writing javascript isn't much about performance anyway...). In the end I think the plugin should work as expected. That's why I brought it up. Others might run into this issue, too.

@shaack Thank you for your feedback. Good luck with fixing.

**Comment by shaack** [comment]
Fixed with 2.0.10. Thanks for reporting. 👍
